gluten-core/src/main/scala/io/glutenproject/substrait/SubstraitContext.scala

```diff
@@ -72,6 +72,9 @@ class SubstraitContext extends Serializable {
   private val operatorToRelsMap = new java.util.HashMap[
     java.lang.Long, java.util.ArrayList[java.lang.Long]]()
 
+  // Only for debug conveniently
+  val operatorToPlanName = new java.util.HashMap[java.lang.Long, java.lang.String]()
```

Comment:
Is it relly  need in the final code？

Reply:
will move `PlanNodesUtil.scala` to ch backend in another pr, which is used only in ch backend.

Reply:
will move `PlanNodesUtil.scala` into ch backend, which is used only in ch backend.

Reply:
keep it in the code, will be used for debug later

Reply:
OK, that looks good to me.